PHPCompatibility/Tests/sniff-examples/argument_functions_report_current_value.php

```diff
@@ -0,0 +1,8 @@
+<?php
+function foo($x) {
+    $x++;
+    var_dump(func_get_arg(0));
+    var_dump(func_get_args);
+    var_dump(debug_backtrace());
+}
+foo(1);
```

Comment:
The `foo(1)` call is not needed.

And what about testing for the use of the function within a closure ?